test/BinaryVotingPowerCalculator.unit.t.sol

```diff
@@ -199,6 +272,175 @@ contract Constructor is BinaryVotingPowerCalculatorTest {
   }
 }
 
+contract GetEarningPower is BinaryVotingPowerCalculatorTest {
+  /// @dev Oracle is available && delegate IS eligible → return sqrt(votingPower)
+  function testFuzz_ReturnsVotingPowerIfOracleIsAvailableAndDelegateIsEligible(
+    address _delegate,
+    uint256 _votingPower
+  ) public {
+    _setOracleIsAvailableAndDelegateIsEligible(_delegate, _votingPower);
```

Comment:
I think the below is easier to grok. Maybe remove one layer of abstraction and use multiple setup methods. I don't see a ton a benefit to these helpers if they are simply wrapping other helpers.

```
    _setOracleAsAvailable();
    _setDelegateeAsEligibleWithVotingPower(_delegate, _votingPower);
```

Reply:
It's basically the setup for the test and it's used in two places. Will remove it for now